src/app/stake/page.tsx

```diff
@@ -5,103 +5,177 @@ import Image from "next/image";
-                      width={29}
-                      height={32}
-                    />
-                  </button>
-                  <button className="bg-yieldi-brown-light text-white  ">
-                    <Image
-                      src="/upload.svg"
-                      alt="Unstake asset"
-                      width={29}
-                      height={28}
-                    />
-                  </button>
-                </Table.Cell>
-              </Table.Row>
+          <Table.Body className="space-y-1.5">
+            {filteredAssets?.map((asset: StakeAsset, index: number) => (
+              <>
+                <Table.Row
+                  key={asset.assetName}
+                  className="mb-[5px] gap-2.5 w-full border border-yieldi-gray-200 bg-white hover:bg-gray-50 [--table-row-box-shadow:none]"
+                  onClick={handleOnClick(asset.assetSymbol)}
+                  data-tooltip-id={`tooltip-${asset.assetName}`}
+                  data-tooltip-content={`View ${asset.assetName} details`}
+                  data-tooltip-place="top"
+                >
+                  <Table.Cell className="p-4 whitespace-nowrap">
+                    <div className="flex items-center ">
+                      <Image
+                        src={`/${asset.assetSymbol.toLocaleLowerCase()}.svg`}
+                        alt={`${asset.assetName} Logo`}
+                        width={50}
+                        height={50}
+                        className="rounded-full pr-2"
+                      />
+                      <span className="flex-col items-center">
+                        <span className="text-yieldi-brown font-gt-america text-xl font-normal">
+                          {asset.assetSymbol}
+                        </span>
+                        <br />
+                        <span className="text-yieldi-brown-light font-gt-america text-sm font-normal">
+                          {asset.assetName}
+                        </span>
+                      </span>
+                    </div>
+                  </Table.Cell>
+                  <Table.Cell className="px-6 py-4 items-center h-full ">
+                    <div className="text-yieldi-brown text-xl font-normal ">
+                      $0.00
+                    </div>
+                    <div className="text-yieldi-brown-light text-sm font-normal">
+                      {isConnected
+                        ? (
+                            satoshiToBtc(btcWalletBalanceSat) * asset.price
+                          ).toString()
+                        : "-"}
+                    </div>
+                  </Table.Cell>
+                  <Table.Cell className="px-6 py-4 ">
+                    <div className="text-yieldi-brown text-xl font-normal flex items-center h-full  ">
+                      {isConnected
+                        ? (
+                            satoshiToBtc(btcWalletBalanceSat) * asset.price
```

Comment:
Remove `* asset.price`. Just need the btc wallet balance

Reply:
done